.eslintrc.js

```diff
@@ -0,0 +1,70 @@
+// Find more about the rules below here https://eslint.org/docs/rules/
+module.exports = {
+  root: true,
+  extends: 'airbnb-base',
+  env: {
+    node: true,
+    es6: true,
+    mocha: true
+  },
+  rules: {
+    'one-var': 0,
+    'one-var-declaration-per-line': 0,
+    'new-cap': [
+      2,
+      {
+        newIsCap: true,
+        capIsNew: true,
+        properties: true
+      }
+    ],
+    'consistent-return': 0,
+    'no-param-reassign': 0,
+    'comma-dangle': 0,
+    curly: ['error', 'multi-line'],
```

Comment:
@nkpremices there's another `curly` rule on line 52. Let's remove one of them.

Reply:
Noted. let me remove one of them